tests/Feature/Api/V1/GameProgressionTest.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use App\Models\Achievement;
+use App\Models\Game;
+use App\Models\PlayerSession;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\Feature\Platform\Concerns\TestsPlayerAchievements;
+use Tests\TestCase;
+
+class GameProgressionTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+    use TestsPlayerAchievements;
+
+    public function testUnknownGame(): void
+    {
+        $this->get($this->apiUrl('GetGameProgress', ['i' => 999999]))
```

Comment:
Typo: this should be `GetGameProgression`.

Reply:
fixed